evap/staff/templates/staff_evaluation_textanswers_section.html

```diff
@@ -11,18 +11,25 @@
                 <table class="table table-striped{% if not forloop.last %} mb-4{% endif %}">
                     <thead>
                         <tr>
-                            <th style="width: 85%">{% trans 'Text answer' %}</th>
+                            <th style="width: 80%">{% trans 'Text answer' %}</th>
+                            <th style="width: 5%"></th>
                             <th style="width: 15%">{% trans 'Publish' %}</th>
                         </tr>
                     </thead>
                     <tbody>
                         {% for answer in result.answers %}
                             <tr id="{{ answer.id }}">
                                 <td class="text-answer">
-                                    <a class="textanswer-link" href="{% url 'staff:evaluation_textanswer_edit' semester.id evaluation.id answer.id %}">{{ answer.answer|linebreaksbr }}</a>
                                     {% if answer.original_answer %}
-                                        <br />
                                         <span class="textanswer-original">({{ answer.original_answer|linebreaksbr }})</span>
+                                        {{ answer.original_answer }}
+                                    {% else %}
+                                        {{ answer.answer }}
+                                    {% endif %}
+                                </td>
+                                <td>
+                                    {% if user.is_manager %}
+                                        <a class="btn btn-sm btn-outline-secondary" href="{% url 'staff:evaluation_textanswer_edit' semester.id evaluation.id answer.id %}"><span class="fas fa-pencil-alt"></a>
```

Comment:
This is not sufficient for the requirement "Editing should only be possible for managers, not for reviewers.". This change hides the functionality of editing from reviewers, they are still permitted to do so if they find the right url though (for example, as the `reviewer` user, I can still visit `staff/semester/21/evaluation/1600/textanswers#176c70ed-0a94-4d34-a51f-c50f307fbdd6` and make the change). The views responsible need to be protected with `@manager_required`.

(a test that tries to access such a view as a reviewer and asserts a failure would be nice)